rest/data/impl.go

```diff
@@ -33,45 +33,10 @@ type DBConnector struct {
 	TaskReliabilityConnector
 	DBCommitQueueConnector
 	SchedulerConnector
+	MockCommitQueueConnector
```

Comment:
I've done this with the caveat being MockDBConnectorImpl is what exists instead of MockCommitQueueConnector since there are a few host / alias related methods that turned out needing to be mocked as well. They're few and far between so I put them in the same implementation for cleanliness.